### ISSUE TITLE
DatePicker : onDateChange callback for Android

### ISSUE THREAD

**Issue by Corpsomium** [GeekyAnts/NativeBase#2016]
Hi,

NativeBase's version v2.5.2 added support for `onDateChange` on iOS (#1987), but I need it on Android.

The support on IOS is made by using an internal `setDate(date)` function. Why not use it on both cases ?

Actual code :
```js
async openAndroidDatePicker() {
/****/
      if (action === "dateSetAction") {
        this.setState({ chosenDate: new Date(year, month, day) });
      }
```

That could turn into something like :
```js
async openAndroidDatePicker() {
/****/
      if (action === "dateSetAction") {
        this.setDate.bind(this)(new Date(year, month, day));
      }
```

Also, the `setDate` and `showDatePicker` functions could be turned into an arrow function to avoid binding on every render, but instead only once at the object's instantiation.

Edit : Oh, and I forgot : I can make a `PR` about it.

**Comment by EricGhildyal** [comment]
Any updates on this? Are there any workarounds for now? I would love for this to be added!

**Comment by Corpsomium** [comment]
Little update on this : I added it on my project and it works just fine.
Maybe not on all cases, but on every that I used, it worked.

**Comment by Corpsomium** [comment]
[Corresponding changed line](https://github.com/Corpsomium/NativeBase/blob/7b39a9165a638746f71c96136579b1af42248121/src/basic/DatePicker.js#L56)
[Full commit with several changes](https://github.com/Corpsomium/NativeBase/commit/7b39a9165a638746f71c96136579b1af42248121)  

I don't get why it would not work on Android ?

**Comment by ferreirix** [comment]
@Corpsomium That's exactly what I suggested when the iOS onDateChange support was added. Maybe we're missing something?

**Comment by SupriyaKalghatgi** [comment]
Fixed with [2.6.1](https://github.com/GeekyAnts/NativeBase/releases/tag/v2.6.1)
